src/message-components/decorators/values.decorator.ts

```diff
@@ -1,9 +1,13 @@
 import { createParamDecorator, ExecutionContext } from '@nestjs/common';
 import { NecordExecutionContext } from '../../context';
 
+/**
+ * @deprecate since v5.4 - old name for `@SelectedStrings`. Will be removed in v6
+ * TODO: Remove in v6
+ */
```

Comment:
Add @see for jsdoc

Reply:
Sure, i will add the link for discordJs and discord-api-types